### ISSUE TITLE
Why not use the WHO International certificate?

### ISSUE THREAD

**Issue by gagarine** [admin-ch/CovidCertificate-Documents#3]
What are the arguments to not use the yellow paper based International Certficate of Vaccination?

I just got a new one for 1.35 CHF and it's bug free.
<img width="301" alt="Screenshot 2021-05-31 at 16 19 16" src="https://user-images.githubusercontent.com/164074/120206887-09945e00-c22c-11eb-8390-6a6c84b4dca2.png">

WHO is also working on a digital version of this certificate https://www.who.int/ihr/ports_airports/icvp/en/

<img width="377" alt="Screenshot 2021-05-31 at 16 20 13" src="https://user-images.githubusercontent.com/164074/120206966-1d3fc480-c22c-11eb-88a0-af70abf2ad99.png">

Their is a working group: https://www.who.int/groups/smart-vaccination-certificate-working-group 

Why Swiss need their own version?  Is is different from the WHO digital one? What are the specific features?

Their is a similar question in the EU parliament: https://www.europarl.europa.eu/doceo/document/P-9-2021-001802_EN.html

More generally about a digital passport, Mozilla nail it: "The question we need to ask ourselves is “is this vaccine passport system going to be empowering, or exclusionary?" https://foundation.mozilla.org/en/blog/digital-covid-vaccine-passports-five-key-takeaways/ . At a global scale, a digital only solution will definitely be exclusionary.

**Comment by ghost** [comment]
We've always gone our own way

**Comment by gagarine** [comment]
> We've always gone our own way

We pretend, struggle, and then apply international norms like any modern country, but we keep pretending doing it all alone. We mostly change the name and eventually the cover picture of the norm.

Anyway in this context "going our own way" mean duplication of effort as you will need an internationally recognized standard to go in foreign countries.

**Comment by micpingpong** [comment]
Hello, on the one hand, temporary certificates based on Covid tests are not issued for vaccinations, and on the other hand, the yellow papers are very easy to forge. regards

**Comment by gagarine** [comment]
@micpingpong the WHO is working with Estonia on a version with a temper-proof QR code so you can verify it (exactly like the confederation proposal). And this is what covid certificates are: a temper-proof QR code generator,  you can display this QR code on paper, a computer or mobile screen, a t-shirt, whatever. Basically it's similar to a QR code based concert ticket...

Now, do we have any evidence of forgery at scale regarding the current yellow paper? I doubt forgery would be more than 0.1%. This was the system used for decades with success in my knowledge. I'm wrong?

Yet, this issue is **not** about digital vs paper (digital information print on paper is digital too), neither about temper-proof QR code based on cryptography or "temper-proof" ink based on signature and ink stamps. It's about having a swiss local system vs a global standard for a covid certificate.

**Comment by chrisleduc** [comment]
There is an overlap but it is not equal. The Covid certificate covers vaccination, immunity through previous infection and recent negative test results.

The digital WHO vaccination certificate in works can be used for travel and might very well be used instead of that for vaccinated passengers. But it does not respond to the needs of passengers that cannot or do not want to be vaccinated for example. 

The solution appears to be developed with the aim to be interoperable with other solutions on a global and European scale. This was repeatedly mentioned by the Swiss government IIRC. 

I hope my understanding is correct. Let me know if I got something wrong.

**Comment by gagarine** [comment]
To answer part of my own question. I read the doc a bit more and the "swiss" covid certificate follow the EU schema: https://github.com/ehn-digital-green-development/ehn-dgc-schema . Is really like I said, we apply international standard.

The stated goal at the EU level is to guarantee the free movement in the Schengen area. 

Edit: @chrisleduc so yes (I saw your post once I posted). It's a EU level digital ID + immunity information. 
Edit2: @chrisleduc I just checked my paper based WHO vaccination certificate, interestingly people that cannot be vaccinated is mentioned. In my opinion, EU is also interested to deploy and test a digital ID system they control, could be useful in the future. 
<img width="693" alt="Screenshot 2021-05-31 at 20 33 05" src="https://user-images.githubusercontent.com/164074/120230364-70773e80-c24f-11eb-8af6-f3492249d58b.png">

**Comment by romanf** [comment]
WHO initiative is good but probably not fast enough. After all, this should be a temporary thing (according to Swiss officials) until at most end of 2021.
Also, the yellow paper can (and is) easy to forge: https://www.rtl.de/cms/gefaelschte-corona-impfpaesse-so-leicht-gibt-s-den-fake-ausweis-rtl-reporter-macht-test-4751655.html (German sorry).

**Comment by ghost** [comment]
Welcome to Switzerland they just follow the EU specs and refuse to put in aditional security elements. Reason its secure enough and this is not in specs.
I dont understand why millions are given out for such a solution, when it could be solved in a better way. 
We can also go back to paper which costs less...

**Comment by romanf** [comment]
What additional security elements are you thinking of?
I think there is no 100% security. So it's always "secure enough". :)

**Comment by informatik-bks** [comment]
I agree with ghost. The project costs are enough to make sure the swiss people became somthing secure if we do it on our own.
I also agree with gagarine. Swiss Gov is giving out a lot of money for something that is already in the pipeline somwhere else why not helping them?
